docker-compose.lando-api.yml

```diff
@@ -26,11 +27,14 @@ services:
       # Use ./Dockerfile to run using uwsgi, similar to remote environments
       dockerfile: ./Dockerfile-dev
     image: suite_lando-api
-    command: ["landing-worker"]
+    command: ["start-landing-worker"]
     environment:
       ENV: localdev
       # TREESTATUS_URL: "http://autoland.treestatus:8000/"
       DATABASE_URL: postgresql://postgres:password@lando-api.db/postgres
+      PHABRICATOR_URL: http://phabricator.test/
+      PHABRICATOR_UNPRIVILEGED_API_KEY: api-lefsv24henzsbzpw337bhizawuyh
+      PHABRICATOR_ADMIN_API_KEY: api-lefsv24henzsbzpw337bhizawuyh
```

Comment:
NOTE: these can all go into a reusable template at the top.

Reply:
Derya Demir

Reply:
Aşksın sen

Reply:
bin/run-all.sh

Reply:
![image](https://user-images.githubusercontent.com/112832178/189776811-a1b3ab18-8e35-4bc8-9ceb-d3ce051a21a8.png)